pygly/WURCS2monoformatter.py

```diff
@@ -0,0 +1,156 @@
+import re
+import os
+import ConfigParser
+from Monosaccharide import *
+
+# This line import the package mutually, be careful
```

Comment:
My suggestion was to only import the part you needed.

from GlycanFormatter import WURCS20ParseError 

and 

class UnsupportedMonoError(WURCS20ParseError):

so that users can trap exceptions with some ability to control them. Did this not work due to circular imports?

Reply:
Yes. I googled a little bit, some of them say that don't use "from ... import ...", instead just "import ...". I tried, but it won't work, it still cause an error.
And some say put the import line to the bottom of the file, and that seems too strange...